dolphin/api/views/storages.py

```diff
@@ -22,5 +24,11 @@ def build_storages(storages):
 
 
 def build_storage(storage):
+    view = {}
     view = copy.deepcopy(storage)
+
+    # Sql Numeric(Decimal) type is not able to be serialized directly into JSON , Need to convert all numeric fields
+    # to float
+    numeric_keys = ['total_capacity', 'used_capacity', 'free_capacity']
+    common.convert_numeric_to_float(numeric_keys, view)
```

Comment:
is it required if user choose mysql or postgres?

Reply:
yes it is , sqlalchemy filed (numeric)